docs/GettingStartedWindows.md

```diff
@@ -39,9 +39,21 @@
 ## Installation
 
 1. Install Visual Studio from [Microsoft](https://visualstudio.microsoft.com).
-2. Install Swift Toolchain from [//swift/build](https://compnerd.visualstudio.com/compnerd/swift-build).
+2. Install Swift Toolchain from either the GitHub [releases](https://github.com/compnerd/swift-build/releases) or the [//swift/build](https://compnerd.visualstudio.com/compnerd/swift-build) CI Pipeline.
 3. Deploy Windows SDK updates.  This must be run from an (elevated) "Administrator" `x64 Native Tools for VS2019 Command Prompt` shell.
 
+Before executing the following, you'll need to ensure that you've set the following values for the build parameter variables: 
+
+* **SystemDrive** 
+    * _Example_: ``` set SystemDrive=C ```  
+* **UniversalCRTSdkDir**
+    * _Example_: ```set UniversalCRTSdkDir=C:\Program Files (x86)\Windows Kits\10\ ```
+* **UCRTVersion**
+    * _Example_: ```set UCRTVersion=10.0.19041.0```
```

Comment:
These will be set for your, you shouldn’t have to set them.  Are you sure that you used the correct command prompt?  It **MUST** be the Native x64 Developer Command Prompt, not Command Prompt.